Simplified/Book/Models/NYPLBookRegistry.h

```diff
@@ -104,11 +106,16 @@ typedef NS_ENUM(NSInteger, NYPLBookState);
 // will overwrite the existing book as if |updateBook:| were called. The location may be nil. The
 // state provided must be one of NYPLBookState and must not be |NYPLBookStateUnregistered|.
 // TODO: Use NYPLBookState instead of NSInteger when migrate to Swift
+// Note: The object type of the audiobook bookmarks array is not specified
+// because using a forward declaration class in the function signature
```

Comment:
You are right, it now compiles without issue. Guess my Xcode was trying to give me a hard time, thanks for catching this.

Reply:
The main build does compile but the unit test doesn't when the function signature includes `NYPLAudiobookBookmark`, because the unit test is in Swift.
After some investigation, since `NYPLAudiobookBookmark` is a forward declaration in ObjC but the `NYPLAudiobookToolkit` might not be imported due to the `FEATURE_AUDIOBOOKS` macro. That would causes any ObjC parameter or function not being compile into the `SimplyE-Swift.h`, and eventually inaccessible from Swift.
So when we try to run the unit tests which does not include `FEATURE_AUDIOBOOKS` macro, it fails to compile.

I think there are two ways to solve the issue here:
1. Use `FEATURE_AUDIOBOOKS` macro for a new function that includes `NYPLAudiobookBookmark`, and the original function remain unchanged.
2. Keep the original function and just keep is as it, currently we do not pass in any bookmarks when calling this function, so it's totally fine to not add `NYPLAudiobookBookmark` to the function signature. And if we need to add bookmark, we can call the other function the designated to add audiobook bookmark.